File: 1512-number-of-good-pairs/1512-number-of-good-pairs.go
func numIdenticalPairs(nums []int) int {
    count := 0
    ln := len(nums)
    // fmt.Println(ln)
    for i := 0; i < ln; i++ {
        for j := i + 1; j < ln; j++ {
            // fmt.Println(nums[i])
            if nums[i] == nums[j] {
                count += 1
            }
        }
    }
    
    return count
}

File: 1512-number-of-good-pairs/1512-number-of-good-pairs.py
class Solution:
    def numIdenticalPairs(self, nums: List[int]) -> int:
        c = 0
        ln = len(nums)
        for i in range(ln):
            for j in range(i+1, ln):
                if nums[i] == nums[j]:
                    c += 1
                    
        return c